apollo-federation/src/operation/mod.rs

```diff
@@ -2919,6 +2978,26 @@ impl SelectionSet {
         Ok(normalized_selections)
     }
 
+    /// Removes the @defer directive from all selections without removing that selection.
+    // PORT_NOTE: This behavior differs from the JS code, which removes anything with @defer.
+    // TODO(@TylerBloom): After the private preview and @defer being impl-ed, check if this logic
+    // should be changed to mirror the JS code.
+    fn without_defer(&mut self) {
+        // TODO: This doesn't seem like the correct way to get the directive name...
+        let Some(defer_name) = self.schema.get_directive_definition(&name!("defer")) else {
+            // TODO: Return an error? Continue? Dunno...
+            return;
+        };
```

Comment:
If we are unable to fetch the `@defer` directive definition, we should return early, ya?